behave/features/cla_fala/fala_end_to_end_tests.feature

```diff
@@ -27,8 +27,8 @@ Feature: FALA end to end tests
     Then the result page containing "<location>" is updated to apply the filter "<filter_label>"
     Examples:
       | location | filter_label |
-      | SW1H9AJ  | Crime        |
-      | London   | Housing      |
+      | SW1H9AJ  | crm          |
```

Comment:
why are these changed?

Reply:
The values of each of the checkbox items are linked to a 3/4 letter string for each of them instead of the readable label itself. 
Example: the label may read "Claims Against Pubic Authorities" but the checkbox input value assigned is "aap" 

FALA also uses this same 3/4 letter string to build the URL and ignores the label. For example the url generated from above step would add "&categories=aap" instead of the nice lovely checkbox. 

 I'm not sure whether this is django or if someone has intentionally designed it this way.

Reply:
Discussed and understood the reason for this.